modules/aws-emr-iam/main.tf

```diff
@@ -718,11 +718,11 @@ resource "aws_iam_role" "emr_ec2_instance_profile" {
   tags                 = var.tags
 }
 
-// The IAM role policy attachment(s) that attach s3 policy ARNs to the EMR EC2 iam role
-resource "aws_iam_role_policy_attachment" "emr_ec2_s3_policies" {
-  count      = length(var.s3_policy_arns)
+//IAM role policy attachment(s) that attach additional policy ARNs to the EMR EC2 iam role
+resource "aws_iam_role_policy_attachment" "emr_ec2_additional_policies" {
+  count      = length(var.additional_policy_arns)
```

Comment:
Should we be using the "effective" approach in here as well? The customer that doesn't update to use `additional_policy_arns` var would be losing the attachments in this case, right?

Reply:
you are righ in here.